src/lib/resources/auth.resource.ts

```diff
@@ -20,12 +28,48 @@ export class AuthResource extends BaseResource {
     try {
       const response = await this.api.auth.isLoggedIn();
       if (response.isSuccessful) {
-        return response.Results[0].data[0].SimpleVO?.value === true;
+        return response.Results[0].data[0].SimpleVO.value === true;
       } else {
         return false;
       }
     } catch (err) {
       return false;
     }
   }
+
+  /**
+   * Set the current client instance to use a given archive when making any subsequent requests.
+   *
+   * *Note: This is done automatically when calling `perm.init()`*
+   *
+   *
+   * #### Example
+   * ```js
+   * const perm = new Permanent(config);
+   * await perm.auth.useArchive(anotherArchiveNbr)
+   * ```
+   *
+   * @returns a Promise that resolves to a boolean representing whether or not the credentials are valid
+   */
+  public async useArchive(archiveNbr: string): Promise<void> {
```

Comment:
Following up from the earlier comment -- what is the scope of the auth resource?  Is it *just* to handle authentication?  If so, can we separate this logic to something else that is specific to archive context management?

If not, should we consider changing the name of this resource to something else?

Reply:
Hmm good point, I guess it should be renamed? it's meant to be a catch all for things related to the session, which includes selecting an archive. Maybe `SessionResource`?

Reply:
Ah perfect yes -- that would obviate my concerns!

Reply:
Great, done!